samples/TempLCD/GroveLCDLib.js

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) 2016, Intel Corporation.
+// JavaScript library for the Grove LCD
+
+export function GroveLCD() {
```

Comment:
Well, this comes from the I2C demo code where the point was to show how to use the I2C API.